pyoptmat/chunktime.py

```diff
@@ -387,7 +377,7 @@ def forward(self, v):
                 v[i] - torch.bmm(self.B[i - 1], v[i - 1].clone()),
```

Comment:
why clone()?

Reply:
It should not be necessary but `torch.autograd` throws an "in place" operation complaint without it.  I'll add a comment.